src/scopes/grants/activeWithin.js

```diff
@@ -57,18 +63,19 @@ export function activeWithinDates(dates) {
         endDate: {
           [Op.gte]: new Date(sd),
         },
-        [Op.or]: [{
-          inactivationDate: {
-            [Op.gte]: new Date(sd),
-          },
-        }, {
-          inactivationDate: null,
-        }],
+        [Op.or]: [
+          { inactivationDate: { [Op.gte]: new Date(sd) } },
```

Comment:
Are these duplicated?

Reply:
No, because we  want either null or gte the provided date:

```ts
  [Op.or]: [
    { inactivationDate: { [Op.gte]: new Date(sd) } },
    { inactivationDate: null },
  ],
```